x-pack/plugins/security_solution/public/common/components/event_details/overview/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { getEnrichedFieldInfo } from '../helpers';
+import {
+  ALERTS_HEADERS_RISK_SCORE,
+  ALERTS_HEADERS_RULE,
+  ALERTS_HEADERS_SEVERITY,
+  SIGNAL_STATUS,
+} from '../../../../detections/components/alerts_table/translations';
+import {
+  SIGNAL_RULE_NAME_FIELD_NAME,
+  SIGNAL_STATUS_FIELD_NAME,
+} from '../../../../timelines/components/timeline/body/renderers/constants';
+import { FormattedFieldValue } from '../../../../timelines/components/timeline/body/renderers/formatted_field';
+import { OverviewCard, OverviewCardWithActions } from '../overview/overview_card';
+import { StatusPopoverButton } from '../overview/status_popover_button';
+import { SeverityBadge } from '../../../../../public/detections/components/rules/severity_badge';
+
+export const NotGrowingFlexGroup = euiStyled(EuiFlexGroup)`
+  flex-grow: 0;
+`;
+
+interface Props {
+  browserFields: BrowserFields;
+  contextId: string;
+  data: TimelineEventsDetailsItem[];
+  eventId: string;
+  handleOnEventClosed: () => void;
+  indexName: string;
+  timelineId: string;
+}
+
+export const Overview = React.memo<Props>(
+  ({ browserFields, contextId, data, eventId, handleOnEventClosed, indexName, timelineId }) => {
+    const statusData = useMemo(() => {
+      const item = find({ field: SIGNAL_STATUS_FIELD_NAME, category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const severityData = useMemo(() => {
+      const item = find({ field: 'kibana.alert.rule.severity', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const riskScoreData = useMemo(() => {
+      const item = find({ field: 'kibana.alert.rule.risk_score', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const ruleNameData = useMemo(() => {
+      const item = find({ field: SIGNAL_RULE_NAME_FIELD_NAME, category: 'kibana' }, data);
+      const linkValueField = find({ field: 'kibana.alert.rule.uuid', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+          linkValueField,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    return (
+      <NotGrowingFlexGroup gutterSize="s">
+        {hasData(statusData) && (
```

Comment:
consider adding some tests for the conditional rendering here, and for the new `overview_card` and `status_popover_button` components in the `overview` directory as the PR progresses